tests/bsim/bluetooth/host/adv/periodic/src/per_adv_advertiser.c

```diff
@@ -228,6 +252,31 @@ static void main_per_adv_advertiser(void)
 	PASS("Periodic advertiser passed\n");
 }
 
+#if defined(CONFIG_BT_CTLR_PHY_CODED)
+static void main_per_adv_advertiser_coded(void)
+{
+	struct bt_le_ext_adv *per_adv;
+
+	common_init();
+
+	create_per_adv_set_coded(&per_adv);
+
+	start_per_adv_set(per_adv);
+	start_ext_adv_set(per_adv);
+
+	/* Advertise for a bit */
+	k_sleep(K_SECONDS(10));
```

Comment:
Ideally we'd replace this sleep with a signal using the BSIM backchannels, but it's not yet commonly used so won't block this PR based on this